### ISSUE TITLE
Tabs container receives visual focus indicator when clicking in the container

### ISSUE THREAD

**Issue by mattrosno** [IBM/carbon-components-angular#770]
https://angular-step-6-carbon-tutorial.netlify.com

Clicking on "What is Carbon?" or anywhere in the tab container that doesn't have its own focus puts visible focus on the tab container, resulting in an unexpected browser focus indication on the tab container. See blue in this screenshot:

![image](https://user-images.githubusercontent.com/1691245/65527067-af1ff780-deb7-11e9-8db0-6e6966a90fcc.png)

This is a discrepancy with the React implementation, e.g.:

https://react-step-6--carbon-tutorial.netlify.com/#/

**Comment by cal-smith** [comment]
This is to follow the W3 tabs aria guidance - https://www.w3.org/TR/wai-aria-practices/examples/tabs/tabs-1/tabs.html

Tab panels should have a `tabindex="0"` to allow assistive technology users the ability to navigate to the panel content. In this case we may be able to use the tabs `isNavigation` property, but we'll still need to make sure the content can be reached.
